.drone.jsonnet

```diff
@@ -0,0 +1,106 @@
+local BuildInfo(version) = {
```

Comment:
Can you explain the need for this file? I see you have a .drone.yaml below so i'm not sure what this does

Reply:
The `.drone.yml` file that you see in this PR was auto generated from this `.drone.jsonnet` file using `drone jsonnet --stream`.